spec/index.html

```diff
@@ -614,7 +614,8 @@ <h3>RDF Version Announcement</h3>
       </li>
     <li><dfn id="dfn-basic" class="export" data-local-lt="basic">Basic conformance</dfn>
       only supports <a data-lt="RDF graph">graphs</a> or <a data-lt="RDF dataset">datasets</a>
-      with <a>triples</a> that do not contain <a>triple terms</a>.</li>
+      with <a>triples</a> that only contain <a data-lt="">basic RDF terms</a> &mdash; i.e.,
```

Comment:
Is the `data-lt=""` intended?

```suggestion
      with <a>triples</a> that only contain <a>basic RDF terms</a> &mdash; i.e.,
```

Reply:
No, I missed to remove that (while testing some permutations). Thanks for the fixes!